xbmc/pvr/epg/EpgChannelData.cpp

```diff
@@ -32,6 +32,7 @@ CPVREpgChannelData::CPVREpgChannelData(const CPVRChannel& channel)
   m_strSortableChannelNumber(channel.ChannelNumber().SortableChannelNumber())
 {
   SetLastWatched(channel.LastWatched());
+  SetClientOrder(channel.ClientOrder());
```

Comment:
As this method only assign the value to the corresponding member, a simple `m_iOrder(channel.ClientOrder())` in the ctor init list is sufficient here.